src/components/layout/header/menu-flyout.tsx

```diff
@@ -20,6 +20,20 @@ const FullscreenOverlay = styled.div<{ visible: boolean }>`
   transition: bottom 0.3s ease-out, visibility 0.3s ease-out;
 `;
 
+const BackgroundOverlay = styled.div<{ visible: boolean }>`
+  position: fixed;
+  z-index: 900;
```

Comment:
You introduced `FLYOUT_Z_INDEX` but here you create another magic number. Maybe you want to hold that in a variable too ?

Reply:
I saw the fix